frontend-react/src/hooks/filters/UseSortOrder.ts

```diff
@@ -15,32 +16,39 @@ interface SortSettingsAction {
 interface SortSettings {
     column: string;
     order: SortOrder;
+    locally: boolean;
 }
 interface SortFilter {
     settings: SortSettings;
     update: Dispatch<SortSettingsAction>;
 }
 
-const sortSettingsReducer = (
+export const sortSettingsReducer = (
```

Comment:
@mreifman Everything SHOULD still be in tact to receive and update sortSettings.order from this filter hook but mind doing a double check for me?

Reply:
This doesn't change any of the behavior around the `order` value exposed by the hook, so no concerns here.

> Also worth considering how we handle the freezing of backend sort when sorting locally. We'd want the data coming back in a predictable way, no? Might be as easy as checking "if locally then DESC else sortSettings.order"

Put another way, we want to avoid superfluous API calls, which would be triggered by changes to `sortSettings.order`. What about tracking the local sort order separately in the hook state to avoid any confusion?

Reply:
Ooo, I like the idea. Might create a bit of complexity in the arrow display logic but I need to go in and fix some of that anyway. Worth the effort!